app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -33,8 +34,9 @@
 
       <org.oppia.android.app.customview.LessonThumbnailImageView
         android:id="@+id/lesson_thumbnail"
-        android:layout_width="match_parent"
-        android:layout_height="0dp"
+        android:layout_width="wrap_content"
+        android:minWidth="280dp"
+        android:layout_height="158dp"
```

Comment:
Why is this `layout_height` fixed? Earlier it was `0dp`, I think keeping fixed height will create issues in smaller devices.

Reply:
Thanks for correcting me, I have updated it to 0dp